server/nodejs/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "dependencies": {
+    "chai": "^4.3.4",
+    "chai-http": "^4.3.0",
+    "express": "^4.17.1",
+    "mocha": "^9.0.1",
+    "nyc": "^15.1.0"
+  },
+  "name": "lea_backery",
+  "version": "1.0.0",
+  "description": "Backend server in node js for dummy bakery",
+  "main": "index.js",
+  "scripts": {
+    "test": "mocha --exit --reporter spec",
```

Comment:
I would add two more scripts: 
1. `start` - to run the application in prod mode
2. `start:dev` or just `dev` to run it with file watcher (like [`nodemon`](https://www.npmjs.com/package/nodemon))